code/examples/cra-kitchen-sink/src/stories/Lifecycle.stories.js

```diff
@@ -1,8 +0,0 @@
-import React from 'react';
-import LifecycleLogger from '../components/LifecycleLogger';
-
-export default {
-  title: 'Lifecycle',
-};
-
-export const Logging = () => <LifecycleLogger />;
```

Comment:
You needed to check the browser console to see these, and half the methods weren't called anyway. Not sure there was a lot of benefit. If we see problems with react lifecycle, let's create a targetted story with a `play()` function to verify things work right.